internal/stack/serverless.go

```diff
@@ -105,6 +107,11 @@ func (sp *serverlessProvider) createProject(settings projectSettings, options Op
 	}
 	project.Endpoints.Fleet = config.Parameters[paramServerlessFleetURL]
 
+	err = project.AddLogstashFleetOutput(sp.kibanaClient)
+	if err != nil {
+		return Config{}, fmt.Errorf("failed to add logstash fleet output: %w", err)
```

Comment:
This error message is the same as in "AddLogstashFleetOutput" , should we just return `err` without wrapping it ? 

```suggestion
		return Config{}, err
```